Norm.Database/Requests/GuildMovieSuggestions.cs

```diff
@@ -115,6 +115,11 @@ public GetMovieSuggestion(string movieSuggestionImdbId, ulong guildId)
                 this.GuildId = guildId;
             }
 
+            public GetMovieSuggestion(string movieSuggestionImdbId, DiscordGuild guild) : this(movieSuggestionImdbId, guild.Id)
+            {
+

```

Comment:
is this supposed to be empty?

Reply:
So, this constructor is redirecting to the other constructor. It's a wrapper over the other one so that I can pass a Discord object instead of the along ID. It's also less prone to incorrect ordering.